ui/app/AppLayouts/Profile/panels/CommunitiesListPanel.qml

```diff
@@ -30,17 +30,19 @@ StatusListView {
         statusListItemTitle.font.pixelSize: 17
         statusListItemTitle.font.bold: true
         subTitle: model.description
-        tertiaryTitle: qsTr("%n member(s)", "", model.members.count)
+        tertiaryTitle: qsTr("%n member(s)", "", (model.members_count || model.members.count))
```

Comment:
I have to say this is quite bad way to change the semantics in order to accommodate for 2 different models. `ListElement` can contain nested elements, so pls adjust the mock model instead.

Reply:
Example:
```qml
ListElement {
  name: "Apple"
  cost: 2.45
  attributes: [
    ListElement { description: "Core" },
    ListElement { description: "Deciduous" }
  ]
}
```

Reply:
@caybro do you know if `model.members` is including the *actual* members of the community or is `members.count` just directly getting the count from the backend? because in the former case, this means that we're are passing ALL the users of the community to this view, which has no need for it, as it only cares for the member count. in such case woudln't it be better to just replace `model.members.count` with `model.members_count` everywhere?

If we do go with this ListElement, then how in the UI can user specify the number of members directly in a number field? for e.g if the user in storybook specifies 100,000 , will the code have to append 100,000 ListElements?